Firebase/Core/FIRLogger.m

```diff
@@ -237,6 +238,7 @@ void FIRLogBasic(FIRLoggerLevel level,
     asl_log(sFIRLoggerClient, NULL, level, "%s", logMsg.UTF8String);
   });
 }
+#pragma clang diagnostic pop
```

Comment:
In order for this to work properly it should be paired with a `#pragma clang diagnostic push` before any changes you make.

See here for a complete example: https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/Example/Tests/GoogleTest/FSTGoogleTestTests.mm#L308